### PR TITLE
Sending rich text emails using TinyMCE editor

### PR DESCRIPTION
**Pull request by RaghavPro**
#156 

Tested rendering on OSX Mail client, Outlook web and Gmail web

TODO
Add support for uploading images from user's end 

screen shot 2016-01-26 at 14 26 27

### CODE REVIEW

File: cleansweep/core/mailer.py [hunk 1]
```diff
@@ -47,9 +46,6 @@ def sendmail(to_address, subject, message, message_html=None, reply_to=None, cc=
     if reply_to:
         headers['Reply-To'] = reply_to
 
-    if message_html:
-        message_html = pynliner.fromString(message_html)
-
```
Comment (anandology) [inline cleansweep/core/mailer.py]: why did you remove this?

This is required for inlining all the CSS that we use in the emails. IIRC, it is required for the emails that we send on various events.

File: cleansweep/forms.py [hunk 2]
```diff
@@ -115,7 +115,7 @@ class SendMailForm(Form):
                     ('contacts', 'All Contacts')
                 ])
     subject = StringField('Subject', validators=[validators.Required()])
-    message = TextAreaField("Message", validators=[validators.Required()])
+    message = TextAreaField("Message", id="email", validators=[validators.Required()])
```
Comment (anandology) [inline cleansweep/forms.py]: please specify the id when rending the field. I think this is not the right place.

File: cleansweep/templates/admin/sendmail.html [hunk 3]
```diff
@@ -1,25 +1,52 @@
...
+        Done! The email with subject <strong>{{ form.subject.data }}</strong> will be sent to selected people
+        shortly.
+    {% else %}
+        <form id="sendmail-form" role="form" method="POST">
+            {{ form.csrf_token }}
+            {{ render_field(form.people) }}
+            {{ render_field(form.subject) }}
+            <dl>
+                <dt>{{ form.message.label }}</dt>
+                <dd>{{ form.message(rows=15) }}</dd>
```
Comment (anandology) [inline cleansweep/templates/admin/sendmail.html]: You can pass id here. 

Also, why are you using `<dl>` here instead of using the `render_field`? You can call it as `render_field(form.message, rows=15, id="email")`. 

Could you change the id to email-message or something. Email sounds like an input for email address.

File: cleansweep/core/mailer.py (hunk 1 above)
Comment (RaghavPro) [inline cleansweep/core/mailer.py]: The editor does that already. It returns the source code with inline css. That's why I removed it.

File: cleansweep/templates/admin/sendmail.html (hunk 3 above)
Comment (RaghavPro) [inline cleansweep/templates/admin/sendmail.html]: Ah! right.

**Comment by anandology** [comment]
Awesome. Looks great to me. merged!
